crates/ruff_benchmark/benches/ty.rs

```diff
@@ -2,12 +2,12 @@
 use ruff_benchmark::criterion;
 use ruff_benchmark::real_world_projects::{InstalledProject, RealWorldProject};
 
+use std::collections::BTreeSet;
```

Comment:
Let's not use `BTreeSet` for indexed files. It won't give us any guarantees because the deterministic ordering goes out of the window in `project.check`, where we process the files concurrently.